financial-connections/src/main/java/com/stripe/android/financialconnections/features/manualentry/ManualEntryScreen.kt

```diff
@@ -96,97 +100,140 @@ private fun ManualEntryContent(
             )
         }
     ) {
+        when (payload) {
+            is Loading, Uninitialized -> LoadingContent()
```

Comment:
The manual entry pane can now stay loading while the session is auto-completed (this will happen on custom manual entry handoff cases)